### DISCUSSION TITLE
En un schemaInstance de tipo file y con array entre 3 elementos no se me visualiza en Data Base

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#215] [4. Bug Reports] [answered]
Links de referencias: 
https://github.com/Cotalker/documentation/discussions/213
https://github.com/Cotalker/documentation/discussions/208

Hola haciendo QA, cuando en mi survey agrego 3 imagenes, éstas se crearan en la base de datos (de forma id), pero al momento de verlo en Data Base me sale cargando (como esperando).

Adjunto:
**Codigo de la property creada**
```
{ 
    "_id" : ObjectId("60afd30cda5b4700084cc6f8"), 
    "subproperty" : [

    ], 
    "users" : [

    ], 
    "isActive" : true, 
    "search" : [
        "691833", 
        "mueble", 
        "botella", 
        "27", 
        "05", 
        "2021", 
        "17", 
        "94", 
        "946", 
        "691833mueble", 
        "mueblebotella27052021170594946"
    ], 
    "name" : {
        "display" : "691833-Mueble Botella", 
        "code" : "mueble_botella_27_05_2021_17:05:94.946"
    }, 
    "propertyType" : "activo_2021", 
    "company" : ObjectId("5a159c135c48965b06140313"), 
    "createdAt" : ISODate("2021-05-27T17:12:44.981+0000"), 
    "modifiedAt" : ISODate("2021-05-27T17:12:45.546+0000"), 
    "__v" : NumberInt(0), 
    "schemaInstance" : {
        "fecha_registro_ingreso" : "2021-05-31", 
        "cadena" : "609d5d462f0eac0008ef42d6", 
        "distrito" : "609d73142f51370008a93e54", 
        "sala" : "609d9445962d1100099f7483", 
        "tipo_activo" : "609ea11b962d110009a05e1c", 
        "fotos_activos" : [
            "60afd2ee798ea396b57a2d23", 
            "60afd2f6798ea356e07a2d26", 
            "60afd2ff798ea317dd7a2d29"
        ], 
        "observaciones_activo" : "tester"
    }
}
```
**SS de database**
![image](https://user-images.githubusercontent.com/61419471/119870092-f8c3af80-beee-11eb-9179-42f3cc02f0f9.png)

Comentarios

**Comentarios:** el code de la property contiene 38 carácteres, por lo que no sería problema

**Actualmente**: dejé configurado como mínimo 1 y máximo 2 y al hacer QA no tuve problemas

**SS QA con otra property**
![image](https://user-images.githubusercontent.com/61419471/119870388-51934800-beef-11eb-8ad6-7c732f0997d8.png)

Gracias :>

**Comment by gmoplaza** [comment]
Hola @NicolasPLG tengo algunas dudas.
1.- Los archivos subidos son a través de bots?
2.- La cantidad máxima de archivos permitida siempre fue dos o lo cambiaste luego?
3.- Cuales son los permisos del usuario?
No sé si el problema es con los archivos o está pasando otra cosa que no permita la correcta visualización.

**Reply by NicolasPLG** [reply]
hola @gmoplaza te respondo

1.- La subo por un bot y mediante un nwr
2.- era de 1 a 5 pero luego la cambie a 2
3.- los permisos que tiene son db-properties-write, admin-properties-write, con los dos read respectivos + el admin-*-write/read

el bot debe tener tambien esos permisos? (el bot tiene bot-default y full admin

Agregando son los mismos permisos que me ayudaste la otra vez para data base :>

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG acabo de subir un formulario con 4 fotos y se ven correctamente, pero si efectivamente hay algo en lo que tienes que tener cuidado, las validaciones de min max del schemaInstance ocurren solo a nivel de UI pero no a nivel de API por lo que puedes agregar más archivos que el máximo.

![image](https://user-images.githubusercontent.com/34345309/120328505-de0b8500-c2b8-11eb-8243-179f253bae97.png)

![image](https://user-images.githubusercontent.com/34345309/120328466-d4821d00-c2b8-11eb-9dba-d5988eb7b79c.png)

**Reply by NicolasPLG** [reply]
hola @guillevalin entiendo, muchas gracias por la aclaración, haré otro QA entonces y configuraré los mínimos y máximos del survey respecto a data base.

Cerraré esta discusión y en el caso que tenga problemas nuevamente hago referencia.

Gracias
